.github/workflows/release-pr.yml

```
@@ -50,7 +50,7 @@ jobs:
 
       - name: Parse release version
         id: get_version
-        run: echo "::set-output name=version::$(echo $GITHUB_REF | sed 's|^refs/heads/release/v?*||g')"
+        run: echo "::set-output bname=version::`echo "${GITHUB_REF}" | sed 's|^refs/heads/release/v?*||g'`"
```

Is the `b` here an intentional addition?

```suggestion
        run: echo "::set-output name=version::`echo "${GITHUB_REF}" | sed 's|^refs/heads/release/v?*||g'`"
```